tests/sentry/models/test_organizationmember.py

```diff
@@ -74,14 +74,15 @@ def test_send_invite_email_customer_domains(self):
     def test_send_sso_link_email(self):
         organization = self.create_organization()
         member = OrganizationMember(id=1, organization=organization, email="foo@example.com")
+        provider = manager.get("dummy")
         with self.options({"system.url-prefix": "http://example.com"}), self.tasks():
-            member.send_invite_email()
+            member.send_sso_link_email("sender@example.com", provider)
```

Comment:
This test was calling the wrong method 🙃

Reply:
Oooooo nooooo